packages/plugin-ext/src/main/node/plugin-localization-backend-contribution.ts

```diff
@@ -0,0 +1,21 @@
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { LocalizationBackendContribution } from '@theia/core/lib/node/i18n/localization-backend-contribution';
+import { PluginDeployer } from '../../common/plugin-protocol';
+import { PluginDeployerImpl } from './plugin-deployer-impl';
+import { Deferred } from '@theia/core/src/common/promise-util';
```

Comment:
Minor: Please don't import from `src`.

```suggestion
import { Deferred } from '@theia/core/lib/common/promise-util';
```